packages/core/admin/admin/src/pages/SettingsPage/components/SettingsNav/index.js

```diff
@@ -60,6 +61,11 @@ const SettingsNav = ({ menu }) => {
                   key={link.id}
                 >
                   {formatMessage(link.intlLabel)}
+                  {link?.tag && (
+                    <Badge marginLeft={1} active>
+                      EE
+                    </Badge>
+                  )}
```

Comment:
Why not add a `flex` span and use gap? margins are very rigid and don't always work with RTL setups.